.infra/index.ts

```diff
@@ -594,7 +594,7 @@ if (!isAdhocEnv) {
       resources: {
         // TODO: adjust resources based on the actual usage
         requests: {
-          cpu: '1',
```

Comment:
Just as an FYI, this is just the requested CPU. There are no limits set, so it wouldn't be throttled

Reply:
<img width="258" alt="Screenshot 2024-05-30 at 18 56 52" src="https://github.com/dailydotdev/daily-api/assets/203558/19361f5d-8096-4e88-b928-7d8a25089408">

Reply:
looks like it is throttled. on my local linux machine it easily eats 4-6 cores.

Reply:
That's interesting
<img width="124" alt="image" src="https://github.com/dailydotdev/daily-api/assets/1681525/9dcdab26-4880-4402-8a54-27eff5d5ff4a">
<img width="823" alt="image" src="https://github.com/dailydotdev/daily-api/assets/1681525/9c5e65d1-88d6-4050-b51b-8c0a5cd47dd4">